src/openzaak/selectielijst/admin_fields.py

```diff
@@ -84,10 +86,16 @@ def get_processtype_readonly_field(url: str) -> str:
 
 
 def get_resultaat_readonly_field(url: str) -> str:
-    resultaat = retrieve_resultaat(url)
-    return f"{resultaat['volledigNummer']} - {resultaat['naam']} - {resultaat['waardering']}"
+    try:
+        resultaat = retrieve_resultaat(url)
+        return f"{resultaat['volledigNummer']} - {resultaat['naam']} - {resultaat['waardering']}"
+    except InvalidURLError:
+        return "De selectielijstklasse is niet afkomstig uit de Referentielijsten API-service"
 
 
 def get_resultaattype_omschrijving_readonly_field(url: str) -> str:
-    omschrijving = retrieve_resultaattype_omschrijvingen(url)
-    return omschrijving["omschrijving"]
+    try:
+        omschrijving = retrieve_resultaattype_omschrijvingen(url)
+        return omschrijving["omschrijving"]
+    except InvalidURLError:
```

Comment:
This line is currently not covered, could you also add a test for this (or test it in the same test you added already?)